packages/grafana-runtime/src/config.ts

```diff
@@ -21,11 +21,17 @@ import {
 
 export interface AzureSettings {
   cloud?: string;
```

Comment:
Should this prop be marked as deprecated?

Reply:
They both are needed. `cloud` is the default cloud, and `clouds` is a list of available clouds.
`cloud` should have been named `defaultCloud` and this is what it is.